core/sdk/src/api/firebolt/fb_capabilities.rs

```diff
@@ -257,7 +257,7 @@ impl<'de> Deserialize<'de> for FireboltPermission {
     }
 }
 
-#[derive(Debug, Clone, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
Is Serialize required only for contract test feature ? If that the case,  add a custom serialize function under contract test feature tag.